### PR TITLE
Bugfix/stable 3 4 0/81 fix type error

### PR DESCRIPTION
**Pull request by jonasraoni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,23 @@
 # QuickSubmit Plugin
 
-- OJS QuickSubmit Plugin
-- Version: (see version.xml)
+[![Build Status](https://travis-ci.com/pkp/quickSubmit.svg?branch=stable-3_4_0)](https://travis-ci.com/pkp/quickSubmit)
+[![OJS compatibility](https://img.shields.io/badge/ojs-3.5-brightgreen)](https://github.com/pkp/ojs/tree/stable-3_5_0)
```
Comment (bozana) [inline README.md:4]: I think here should be stable-3_4_0 ?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
+[![GitHub release](https://img.shields.io/github/v/release/pkp/quickSubmit?include_prereleases&label=latest%20release)](https://github.com/pkp/quickSubmit/releases)
```
Comment (bozana) [inline README.md:5]: Is there a way to find out the last release for this branch?

File: README.md (hunk 1 above)
Comment (jonasraoni) [reply README.md:4]: Yep! I've cherry-picked from main and didn't update this line and the one below, please update for me, I don't have write permissions here :)

File: README.md (hunk 2 above)
Comment (jonasraoni) [reply README.md:5]: That's right, you can replace by this:
https://img.shields.io/github/v/release/pkp/quickSubmit?include_prereleases&sort=semver&filter=*3.4.0*&display_name=release

Comment (bozana) [reply README.md:5]: OK, I first now see your comment above. OK I will try to do that and we need to maybe then always use the version number in the release name?

Comment (jonasraoni) [reply README.md:5]: Feel free to drop, it's not a big deal for me :)

**Pull request merged**